src/main/java/moves/Move.java

```diff
@@ -0,0 +1,48 @@
+package moves;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import enums.MoveAttribute;
+import regions.District;
+import regions.Precinct;
+
+/**
+ *
+ * @author Jayson
+ */
+public class Move {
```

Comment:
Does the move itself determine the success or failure attribute?  If so the inclusion of a method for determining that should be included in the move class as a singleton class created here.

Reply:
Each Algorithm object will have an objective function calculated after each move. If the function increases the move is kept and recorded as a success, if the function decreases the move is removed and recorded as a failure.

Reply:
Good